cg/apps/demultiplex/sample_sheet/models.py

```diff
@@ -2,44 +2,48 @@
 from pydantic import BaseModel, Field
 from typing import List
 
+from cg.constants.constants import GenomeVersion
 from cg.constants.demultiplexing import SampleSheetHeaderColumnNames
 
 LOG = logging.getLogger(__name__)
 
 
-class NovaSeqSample(BaseModel):
+class FlowCellSample(BaseModel):
     """This model is used when parsing/validating existing sample sheets."""
 
-    flow_cell: str = Field(..., alias=SampleSheetHeaderColumnNames.FLOW_CELL_ID.value)
-    lane: int = Field(..., alias="Lane")
-    sample_id: str = Field(..., alias="SampleID")
-    reference: str = Field(..., alias="SampleRef")
+    flowcell_id: str = Field(..., alias=SampleSheetHeaderColumnNames.FLOW_CELL_ID.value)
```

Comment:
This one is a bit tricky because for tests it reads from [`/tests/fixtures/apps/demultiplexing/raw_lims_samples/raw_samplesheet_novaseq.json`](https://github.com/Clinical-Genomics/cg/blob/master/tests/fixtures/apps/demultiplexing/raw_lims_samples/raw_samplesheet_novaseq.json) and it only works with that name. I am not sure if reading from such a file happens in production but I therefore decided not to touch it. What do you think @henrikstranneheim?